releasenotes/notes/0.25/enable_target_aware_meas_map-0d8542402a74e9d8.yaml

```diff
@@ -12,5 +12,5 @@ features:
      
     qubits 0 and 1, and 2, 3 and 4 are measured together on the device.
     This constraint doesn't block measuring an individual qubit,
-    however, Qiskit scheduler must consider the alignment of 
-    measure operations for those qubits.    
\ No newline at end of file
+    but the Qiskit scheduler must consider the alignment of 
```

Comment:
I would probably reference the scheduler tutorial here. Also I would mentioning the transpiler scheduling stage as it also should take into account the concurrent measurements attribute in the target. The overload of the term "schedule" is confusing here, the `scheduler` (with the entrypoint `schedule()`) builds pulse schedule object (I think it's still `Schedule`, not a `ScheduleBlock` ) from a circuit. While the scheduling stage "schedules" a circuit by inserting delays (or other gates to account for the timing in a circuit. I've seen this trip up a lot of people in the past (I have no idea why, it's not like I wrote "schedule" 10 different times with different meaning each time) so it'd be good to be explicit here.